lib/ssh_scan/api_db.rb

```diff
@@ -0,0 +1,61 @@
+require 'sqlite3'
+
+module SSHScan
+  class APIDatabaseHelper
```

Comment:
I'll probably propose that we do this in the future PR, just wanted to express the idea here.